migration/idempotent/001-base.sql

```diff
@@ -2099,6 +2131,8 @@ BEGIN
     SELECT drop_point OPERATOR(pg_catalog.+) pg_catalog.interval '1 hour'
     INTO check_time;
 
+    SELECT current_epoch FROM _prom_catalog.ids_epoch FOR SHARE INTO STRICT locally_observed_epoch;
```

Comment:
Even though `FOR SHARE` is sufficient here and is what I suggested in the spec, keeping our mxid struggles in mind, I would perhaps upgrade it to `FOR UPDATE`. I don't expect a ton of concurrency on this lock, but it should be easier on IO and vacuum.

Reply:
Done.